test/compiler_test.js

```diff
@@ -87,14 +93,46 @@ describe('Compiler Tests', () => {
                 .setGlobal(payload);
 
               // eslint-disable-next-line import/no-dynamic-require,global-require
-              const service = require(copiledFilename);
+              const service = require(compiledFilename);
               service(runtime).then(() => {
                 const output = runtime.stream;
                 assert.equal(output, test.output);
                 done();
               }).catch(done);
             });
           }
+
+          if ('mappedOutput' in test) {
+            const mapFilename = `${compiledFilename}.map`;
+            it(`${idx}. Maps lines for '${test.name}' correctly.`, (done) => {
+              fs.readFile(mapFilename, 'utf8', (err1, map) => {
```

Comment:
I would use `readFileSync()`. or we could switch to `fs-extras` and use `async/await`. but just for the tests, it's not so a big issue, I think.